### PR TITLE
libxslt, mesa, xwayland: update packages, fix GLX not working, fix font data directories

### PR DESCRIPTION
**Pull request by ElectrodeYT**

### CODE REVIEW

**Review by no92 — APPROVED** [review]
LGTM

File: bootstrap.d/dev-libs.yml [hunk 1]
```diff
@@ -1083,24 +1083,20 @@ packages:
...
-      tools_required:
-        - host-autoconf-v2.69
-        - host-automake-v1.15
-        - host-libtool
-        - host-pkg-config
+      url: 'https://download.gnome.org/sources/libxslt/1.1/libxslt-1.1.38.tar.xz'
+      format: 'tar.xz'
+      checksum: blake2b:9397be8b223bd68b326b954fe6a749a269e969b4669747678715728f553916c23475232fc2894a5e4230e15b031981423adeb8f83407f4db8075a784c7e6a55b
+      extract_path: 'libxslt-1.1.38'
+      version: '1.1.38'
```
Comment (avdgrinten) [inline bootstrap.d/dev-libs.yml:1090]: Core libs should continue to use VCS sources.

**Review by avdgrinten — CHANGES_REQUESTED** [review]
NAK b

File: bootstrap.d/sys-apps.yml [hunk 2]
```diff
@@ -555,19 +555,11 @@ packages:
...
-      regenerate:
-        - args: ['./bootstrap']
-        - args: ['cp',
-            '@BUILD_ROOT@/tools/host-automake-v1.15/share/automake-1.15/config.sub',
-            '@THIS_SOURCE_DIR@/build-aux/']
+      url: 'https://download-mirror.savannah.gnu.org/releases/man-db/man-db-2.12.0.tar.xz'
+      format: 'tar.xz'
+      checksum: blake2b:a6258c1fc9cd81be5d288298660f5b9bda22d726ef98dd5c0a1998809a32391f7244be6897ee8e03483e705a426d6a59d442de3dc2e84f45500daa825ac639db
+      extract_path: 'man-db-2.12.0'
+      version: '2.12.0'
```
Comment (avdgrinten) [inline bootstrap.d/sys-apps.yml:562]: Likewise.

File: bootstrap.d/dev-libs.yml (hunk 1 above)
Comment (ElectrodeYT) [reply bootstrap.d/dev-libs.yml]: Converted to VCS and CMake.

File: bootstrap.d/sys-apps.yml (hunk 2 above)
Comment (ElectrodeYT) [reply bootstrap.d/sys-apps.yml]: Update dropped due to autoreconf problems.

File: bootstrap.d/dev-libs.yml [hunk 3]
```diff
@@ -1097,28 +1097,36 @@ packages:
             'NOCONFIGURE': 'yes'
     tools_required:
       - system-gcc
+      - virtual: pkgconfig-for-target
+        triple: "@OPTION:arch-triple@"
```
Comment (Dennisbonke) [inline bootstrap.d/dev-libs.yml:1101]: Missing tool host-cmake.

File: bootstrap.d/dev-libs.yml [hunk 4]
```diff
@@ -1084,8 +1084,8 @@ packages:
     source:
       subdir: ports
       git: 'https://gitlab.gnome.org/GNOME/libxslt.git'
-      tag: 'v1.1.34'
-      version: '1.1.34'
+      tag: 'v1.1.38'
+      version: '1.1.38'
       tools_required:
```
Comment (Dennisbonke) [inline bootstrap.d/dev-libs.yml:1089]: Autotools tools still needed on cmake? Unlikely.

File: bootstrap.d/media-libs.yml [hunk 5]
```diff
@@ -799,8 +799,8 @@ packages:
     source:
       subdir: 'ports'
       git: 'https://gitlab.freedesktop.org/mesa/mesa.git'
-      tag: 'mesa-23.0.2'
-      version: '23.0.2'
+      tag: 'mesa-23.1.9'
+      version: '23.1.9'
       tools_required:
```
Comment (Dennisbonke) [inline bootstrap.d/media-libs.yml:804]: This seems erroneous. Can you remove this?

**Review by Dennisbonke — CHANGES_REQUESTED** [review]
A few points that need fixing

File: bootstrap.d/dev-libs.yml (hunk 3 above)
Comment (ElectrodeYT) [reply bootstrap.d/dev-libs.yml:1092]: Done

File: bootstrap.d/media-libs.yml (hunk 5 above)
Comment (ElectrodeYT) [reply bootstrap.d/media-libs.yml]: Was indeed erroneous, removed

**Review by Dennisbonke — APPROVED** [review]
LGTM

**Pull request merged**